[2025-02-18T03:00:07.284+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T03:00:07.304+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: default_args_dag.task2 scheduled__2025-02-18T02:00:00+00:00 [queued]>
[2025-02-18T03:00:07.313+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: default_args_dag.task2 scheduled__2025-02-18T02:00:00+00:00 [queued]>
[2025-02-18T03:00:07.313+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T03:00:07.325+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task2> on 2025-02-18 02:00:00+00:00
[2025-02-18T03:00:07.328+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=11967) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-18T03:00:07.329+0000] {standard_task_runner.py:72} INFO - Started process 11968 to run task
[2025-02-18T03:00:07.330+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'default_args_dag', 'task2', 'scheduled__2025-02-18T02:00:00+00:00', '--job-id', '997', '--raw', '--subdir', 'DAGS_FOLDER/12_default_args_dag.py', '--cfg-path', '/tmp/tmpzoxi2mnx']
[2025-02-18T03:00:07.332+0000] {standard_task_runner.py:105} INFO - Job 997: Subtask task2
[2025-02-18T03:00:07.371+0000] {task_command.py:467} INFO - Running <TaskInstance: default_args_dag.task2 scheduled__2025-02-18T02:00:00+00:00 [running]> on host ab74d4d90d32
[2025-02-18T03:00:07.448+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='default_args_dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T02:00:00+00:00'
[2025-02-18T03:00:07.449+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T03:00:07.461+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-18T03:00:07.462+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-18T03:00:07.469+0000] {subprocess.py:99} INFO - Output:
[2025-02-18T03:00:12.471+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-18T03:00:12.496+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T03:00:12.496+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=default_args_dag, task_id=task2, run_id=scheduled__2025-02-18T02:00:00+00:00, execution_date=20250218T020000, start_date=20250218T030007, end_date=20250218T030012
[2025-02-18T03:00:12.558+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T03:00:12.580+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-18T03:00:12.582+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
